### PR TITLE
Upgrade homepage to v1

### PR DESCRIPTION
**Pull request by barrymcgee**
Note: All other template pages will be touched and proposed section by section.

## Done

Upgrade homepage to Vanilla v1 styling

## QA

- Pull code
- Run `./run serve --watch`
- View homepage at http://0.0.0.0:8002/
- Check template code

## Issue / Card

Fixes https://www.github.com/ubuntudesign/vanilla-squad/issues/26

## Demo

http://www.canonical.com-homepage-polish.demo.haus/

### CODE REVIEW

File: templates/500.html [hunk 1]
```diff
@@ -1,237 +1,105 @@
...
+    </div>
+  </footer>
+  <!-- cookie policy -->
+  <script src="/static/js/vendor/cookie-policy.js"></script>
+  <!-- global nav -->
+  <script src="/static/js/vendor/global.js"></script>
+  <script>
+    // Every page should show the cookie policy until accepted
+    var cookie_options = {
+      'content': 'We use cookies to improve your experience of ubuntu.com. By continuing to explore without changing your settings, you are agreeing to accept them. To learn how to change these settings, please see our <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies">privacy policy</a>.'
```
Comment (anthonydillon) [inline templates/500.html]: s/ubuntu.com/canonical.com

File: templates/_base/base.html [hunk 2]
```diff
@@ -1,130 +1,112 @@
...
+
+  <link type="text/plain" rel="author" href="{% versioned_static 'files/humans.txt' %}" />
+
+  <!-- google fonts -->
+  <link href='https://fonts.googleapis.com/css?family=Ubuntu:400,300,300italic,400italic,700,700italic|Ubuntu+Mono' rel='stylesheet' type='text/css' />
+  <link rel="stylesheet" type="text/css" media="screen" href="{{ ASSET_SERVER_URL }}a722db6e-global.css" />
+  <link rel="stylesheet" type="text/css" media="screen" href="{% versioned_static 'css/styles.min.css' %}" />
+
+  <!-- modernizr -->
+  <script src="{{ STATIC_URL }}js/vendor/modernizr.2.7.1.js"></script>
```
Comment (anthonydillon) [inline templates/_base/base.html]: Does this exists?

File: templates/_base/base.html [hunk 3]
```diff
@@ -1,130 +1,112 @@
...
+
+  <!-- cookie policy -->
+  <script src="{{ STATIC_URL }}js/vendor/cookie-policy.js"></script>
+
+  <!-- global nav -->
+  <script src="{{ STATIC_URL }}js/vendor/global.js"></script>
```
Comment (anthonydillon) [inline templates/_base/base.html]: Shouldnt these be in the footer?

File: templates/_base/base.html [hunk 4]
```diff
@@ -1,130 +1,112 @@
...
+    }
+  </script>
+
+  <script src="{{ STATIC_URL }}js/core.js"></script>
+
+  <script>
+
+    // Every page should show the cookie policy until accepted
+    var cookie_options = {
+      'content': 'We use cookies to improve your experience of ubuntu.com. By continuing to explore without changing your settings, you are agreeing to accept them. To learn how to change these settings, please see our <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies">privacy policy</a>.'
```
Comment (anthonydillon) [inline templates/_base/base.html]: s/ubuntu.com/canonical.com

File: templates/_base/base.html (hunk 3 above)
Comment (anthonydillon) [inline templates/_base/base.html]: Otherwise they should be async

**Comment by anthonydillon** [comment]
## QA issues:
- Failed to run `./run` bower packages not added.
- Need to add `.bowerrc` and `bower.json`

File: templates/_base/base.html (hunk 2 above)
Comment (barrymcgee) [inline templates/_base/base.html]: Yep

**Comment by anthonydillon** [comment]
You still need to `bower install canonical-cookie-policy --save`

**Comment by yaili** [comment]
Design notes:
- [ ] Can the hero paragraph be heading-5?
- [ ] And make that block of text one col narrower
- [ ] Link in hero should be inverted
- [ ] Is the empty gap at the top where the global nav will be, or a bug?
- [ ] We need to decide if the nav can stay light grey - maybe to discuss with Ale/Peter?
- [ ] Add divider below We deliver
- [ ] The u-equal-height is adding margin bottom in "We help organisations" and in "We need big thinkers", which makes top/bottom padding not be the same in the that strip
- [ ] Headings in footer should be muted heading
- [ ] In S screen, the images in the rows should be horizontally centred, see live

**Comment by barrymcgee** [comment]
@yaili Yes, empty gap is where global-nav goes - being addressed separately.

@pmahnke What do you think of light navigation for canonical.com? http://www.canonical.com-homepage-polish.demo.haus

**Comment by yaili** [comment]
- [ ] Muted headings shouldn't be centred in footer

**Comment by barrymcgee** [comment]
@yaili Is text being centered not inherent to muted headings pattern, it's one of only three rules that make up the pattern?

**Comment by anthonydillon** [comment]
The design points are not for this phase. We need to capture them for the next phase and fix them in the next iteration.

**Comment by anthonydillon** [comment]
We need to capture these design points in the next iteration.

**Comment by yaili** [comment]
@barrymcgee no, it should be left aligned by default https://github.com/ubuntudesign/vanilla-brochure-theme-design/blob/master/Muted%20heading/muted-heading.md

**Pull request merged**